next.config.js

```diff
@@ -1,21 +1,26 @@
 module.exports = {
-	webpack: (config, { isServer }) => {
-	  // Fixes npm packages that depend on `fs` module
-	  if (!isServer) {
-		config.node = {
-		  fs: 'empty',
-		  module: "empty"
-		}
-	  }
+  // Agility CDN domain config for Next/Image
+  images: {
+    domains: [`${process.env.AGILITY_GUID}-cdn.agilitycms.cloud`],
```

Comment:
@joshua-isaac we'll need this starter to support both CDN URLs we now have to support.

For example: https://cdn.aglty.io/pwmnh3al/agilitycms.png

`https://cdn.aglty.io/{websiteName}/{filePath}`